.bazeliskrc

```diff
@@ -0,0 +1 @@
+USE_BAZEL_VERSION=3.7.2
```

Comment:
Tensorflow uses 4.2.1 now.

Reply:
Thanks, lets merge this as CI fix and then can make separate PR to increase bazel version